www/templates/diary/untracked_time_list_item.html

```diff
@@ -0,0 +1,24 @@
+<div>
+    <ion-item class="unified-diary-item" style="background-color: transparent;" class="list-item">
+        <div class="diary-card short greyed" style="padding-top: 10px;">
+            <div style="width: 100%; margin: auto; padding-block: 5px">
+                <div class="row place-title">
+                    <b translate=".untracked-time-range"
+                        translate-value-start="{{ triplike.display_start_time }}"
+                        translate-value-end=" {{ triplike.display_end_time }}">
```

Comment:
Given the comments from UNSW, we should probably add the start/end times for this as well.

Reply:
The start and end times are listed here in bold text. Do you think it is necessary that they are moved to the badges?

We decided a while ago for places that it would be visually redundant.

Reply:
Yes, we did. And UNSW disagreed 😄 
https://github.com/e-mission/e-mission-docs/issues/871#issuecomment-1486148982